google-cloud-pubsub/lib/google/cloud/pubsub/project.rb

```diff
@@ -431,6 +432,46 @@ def subscriptions token: nil, max: nil
         alias_method :find_subscriptions, :subscriptions
         alias_method :list_subscriptions, :subscriptions
 
+
+        ##
+        # Retrieves a list of snapshots for the given project.
+        #
+        # @param [String] token A previously-returned page token representing
+        #   part of the larger set of results to view.
+        # @param [Integer] max Maximum number of snapshots to return.
+        #
+        # @return [Array<Google::Cloud::Pubsub::Snapshot>] (See
+        #   {Google::Cloud::Pubsub::Snapshot::List})
+        #
+        # @example
+        #   require "google/cloud/pubsub"
+        #
+        #   pubsub = Google::Cloud::Pubsub.new
+        #
+        #   snapshots = pubsub.snapshots
+        #   snapshots.each do |snapshot|
+        #     puts snapshot.name
+        #   end
+        #
+        # @example Retrieve all snapshots: (See {Snapshot::List#all})
+        #   require "google/cloud/pubsub"
+        #
+        #   pubsub = Google::Cloud::Pubsub.new
+        #
+        #   snapshots = pubsub.snapshots
+        #   snapshots.all do |snapshot|
+        #     puts snapshot.name
+        #   end
+        #
+        def snapshots token: nil, max: nil
+          ensure_service!
+          options = { token: token, max: max }
+          grpc = service.list_snapshots options
+          Snapshot::List.from_grpc grpc, service, max
+        end
+        alias_method :find_snapshots, :snapshots
```

Comment:
`alias_method` is a method that accepts symbols and strings as arguments. `alias` is a keyword.

http://blog.bigbinary.com/2012/01/08/alias-vs-alias-method.html